src/interface/TWGroestlcoinAddress.cpp

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "../Base58.h"
+#include "../Groestlcoin/Address.h"
+
+#include <TrezorCrypto/ecdsa.h>
+#include <TrustWalletCore/TWGroestlcoinAddress.h>
+#include <TrustWalletCore/TWPublicKey.h>
+
+#include <cstring>
+
+using namespace TW::Groestlcoin;
+
+bool TWGroestlcoinAddressEqual(struct TWGroestlcoinAddress lhs, struct TWGroestlcoinAddress rhs) {
```

Comment:
could you please implement `TWGroestlcoinAddress` like `TWBech32Address`? We have a issue for reimplementing `TWBitcoinAddress` (https://github.com/TrustWallet/wallet-core/issues/217)